hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/NewHoodieParquetFileFormatUtils.scala

```diff
@@ -150,6 +150,12 @@ class NewHoodieParquetFileFormatUtils(val sqlContext: SQLContext,
       sqlContext.getConf(config.key(), defaultValueOption.getOrElse(config.defaultValue())))
   }
 
+  private def checkIfAConfigurationEnabled(config: ConfigProperty[java.lang.Boolean],
```

Comment:
maybe move this to `ConfigUtils`?

Reply:
I will refactor this into HoodieReaderConfig.